Goldleaf/Program/RomFs/Language/Strings/de.json

```diff
@@ -159,7 +159,7 @@
     "Diese NSP wurde empfangen:",
     "Soll diese installiert werden?",
     "Durchschnittliche Geschwindigkeit:",
-    "Die USB Installation erfolgreich durchgeführt. Der PC Client kann geschlossen werden.",
```

Comment:
Kann man machen, braucht man nicht.
You can do this, but "wurde" is not needed necessary.

Reply:
Doesn't seem to be valid german to me without the "wurde", I'd remove the "Die" or add the "wurde".

Reply:
Yeah, best would be to remove "Die" at the beginning (sorry had a long day).

Reply:
Looks better now 👍